redis/utils.py

```diff
@@ -110,3 +116,11 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+def get_lib():
```

Comment:
I think get_lib_version is more correct name because it is more informative WDYT?